integration/docker/csi/.gitignore

```diff
@@ -0,0 +1 @@
+csi
```

Comment:
Why do we add this?

Reply:
We don't want to add this binary into git repository, so just ignore it